src/keysignote.ts

```diff
@@ -25,7 +25,7 @@ export class KeySigNote extends Note {
     this.setPreFormatted(true);
     this.keySignature.setStave(this.checkStave());
     this.keySignature.format();
```

Comment:
You can remove this (since `getWidth` calls `format`).

Reply:
done